tests/phpunit/bootstrap.php

```diff
@@ -24,7 +24,20 @@
 } elseif ( file_exists( '/tmp/wordpress-tests-lib/includes/bootstrap.php' ) ) {
 	$_test_root = '/tmp/wordpress-tests-lib';
 } else {
-	$_test_root = dirname( dirname( dirname( dirname( TESTS_PLUGIN_DIR ) ) ) ) . '/tests/phpunit';
+	if ( ! getenv( 'WP_PHPUNIT__DIR' ) ) {
+		printf( '%s is not defined. Run `composer install` to install the WordPress tests library.' . "\n", 'WP_PHPUNIT__DIR' );
+		exit;
+	}
+
+	$_test_root = getenv( 'WP_PHPUNIT__DIR' );
```

Comment:
I was surprised to find that this works without requiring the autoloader in here first, but I found that PHPUnit does it automatically in its "entrypoint" (`vendor/phpunit/phpunit/phpunit`). It might be a good idea to load it anyways to support a non-composer installed version of PHPUnit (e.g. a phar). It doesn't cost anything to `require_once` it more than once.